Whitespace changes broke the build

Enter:
    ./configure && make clean && make

./test/patest1.c:129:16: error: incompatible pointer types passing 'FILE *' (aka 'struct __sFILE *') to parameter of type 'const char *'
      [-Werror,-Wincompatible-pointer-types]
        printf(stderr,"Error: No input default device.\n");

Bug introduced in PR #417
This slipped through CI.

--- a/test/patest1.c
+++ b/test/patest1.c
@@ -126,7 +126,7 @@ int main(int argc, char* argv[])
 
     inputParameters.device = Pa_GetDefaultInputDevice();    /* default input device */
     if (inputParameters.device == paNoDevice) {
-        fprintf(stderr,"Error: No input default device.\n");
+        fprintf(stderr, "Error: No input default device.\n");
         goto done;
     }
     inputParameters.channelCount = 2;                       /* stereo input */

--- a/test/patest1.c
+++ b/test/patest1.c
@@ -126,7 +126,7 @@ int main(int argc, char* argv[])
 
     inputParameters.device = Pa_GetDefaultInputDevice();    /* default input device */
     if (inputParameters.device == paNoDevice) {
-        printf(stderr,"Error: No input default device.\n");
+        fprintf(stderr,"Error: No input default device.\n");
         goto done;
     }
     inputParameters.channelCount = 2;                       /* stereo input */

